### ISSUE TITLE
Syllable doesn't have a syllable element (rodan-staging)

### ISSUE THREAD

**Issue by JoyfulGen** [DDMAL/MEI_encoding#30]
While processing folio 083r (Salzinnes), I came across this syllable which seems to exist, but doesn't have a bounding box or a little diamond in the text info section. 
<img width="1402" alt="083r syllable no syl (staging)" src="https://user-images.githubusercontent.com/83373378/124791831-6ca79d80-df1a-11eb-83f1-551bf4f3d0ae.png">

Looking at the MEI file, it looks like the syllable has a neume component, but no syl element:
[083r syllable no BB (staging).mei.zip](https://github.com/DDMAL/MEI_encoding/files/6778559/083r.syllable.no.BB.staging.mei.zip)

This is similar to https://github.com/DDMAL/MEI_encoding/issues/29, but in that issue there IS a syllable element, but no syllable in it, and there are no neume components. Here there ARE neume components, but not syl element at all:

```</syllable>
                                <syllable xml:id="m-718d11f1-bcf1-4901-bfcf-1aeee25c75f6">
                                    <neume xml:id="m-388b7301-4cd3-482c-8b4f-9011a2a5d84e">
                                        <nc xml:id="m-7413ef59-b50b-43f3-8a17-e39d64f00d02" facs="#m-aba18645-ab56-4c8f-a9f9-440e2ca4532d" oct="2" pname="a"/>
                                    </neume>

**Comment by JoyfulGen** [comment]
It happened again, in folio 081r:
<img width="1376" alt="081r no syl (staging)" src="https://user-images.githubusercontent.com/83373378/124806681-f7909400-df2a-11eb-836b-61347d511942.png">

I uploaded the uncorrected file back into rodan2 and it looks like the issue doesn't exist there. Also, I had never come across it before working on rodan-staging. It's quite possible therefore that the issue only exists in staging. This isn't a rock-hard conclusion however, because I had corrected the entirety of the folio in staging before noticing this syl-less syllable, so something might have happened during the correction.
